### ISSUE TITLE
Mislabeled items on the fashion-MNIST dataset

### ISSUE THREAD

**Issue by Shuyib** [keras-team/keras#9487]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ x ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [ x ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ x ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ x ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Hi, i was trying to classify items in the fashion-MNIST dataset. Then i noticed that some labels don't do not match the ones presented here [https://keras.io/datasets/#fashion-mnist-database-of-fashion-articles](url). I explore that in this notebook [https://www.floydhub.com/shuyib/projects/version_3_fashiondb/5/code/fashion-MNIST-CNN.ipynb.](url) and here's my gist https://gist.github.com/Shuyib/3cc4b59b8eb58513a3c3f887694bda59.

NB: I mislabeled the y axis of the line graph with a green line and points. It's supposed to be accuracy not loss.
